modules/sand/src/main/ts/ephox/sand/api/PlatformDetection.ts

```diff
@@ -12,13 +12,15 @@ export type DeviceType = DeviceTypeCore;
 
 const mediaMatch = (query: string) => window.matchMedia(query).matches;
 
-const platform = Cell<PlatformDetection>(PlatformDetection.detect(navigator.userAgent, mediaMatch));
+// IMPORTANT: Must be in a thunk, otherwise rollup thinks calling this immediately
+// causes side effects and won't tree shake this away
+const platform = Cell<() => PlatformDetection>(Thunk.cached(() => PlatformDetection.detect(navigator.userAgent, mediaMatch)));
```

Comment:
I'm all for that, only used a `Cell` as I thought that was the required code style. I'll change it to a `let`, thanks! :+1:

Reply:
It often seems like a required code style, and we perhaps over-use it for that reason, but it's actually _more_ dangerous than a `let` because it shares mutation between modules.